firebase/functions/callable/createAdmin.ts

```diff
@@ -0,0 +1,28 @@
+import { functions } from '../main'
+import { getAuth, Auth, UserRecord } from 'firebase-admin/auth'
+import { getFirestore, Firestore } from 'firebase-admin/firestore'
+
+const auth: Auth = getAuth()
+const firestore: Firestore = getFirestore()
+
+export const createAdmin = functions
+  .region('australia-southeast1')
```

Comment:
This might be a bad take. I reckon that using environment variable in this case would make the code less readable. Moreover, the `region` function is already strongly typed as in the region string name must match exactly in order for the function to even run.

Reply:
We will use your method when we switch back to firebase functions to keep all functions consistent.